src/firebase/auth/functions/index.js

```diff
@@ -641,13 +647,24 @@ async function sendForecastEmails(groupedFavorites) {
         ...emojiAttachments,
         ...emailAttachements,
       ];
+
+      //randomly choose a favorite
+      const randomFavorite = userFavorites[Math.floor(Math.random() * userFavorites.length)];
+
+      const responseFromGetMeasurements = await axios.get(`${process.env.PLATFORM_BASE_URL}/api/v2/devices/measurements/sites/${randomFavorite.placeId}?token=${process.env.API_TOKEN}`,);
+      if (responseFromGetMeasurements.status !== 200) {
+        functions.logger.error("Error getting Measurements", responseFromGetMeasurements);
+        return null;
+      }
+      const pmValue = responseFromGetMeasurements.data.measurements[0].pm2_5.value;
+
       const mailOptions = {
         from: {
           name: "AirQo Data Team",
           address: process.env.MAIL_USER,
         },
         to: userFavorites[0].userEmail,
-        subject: "Exciting Updates from Your Favorite Locations!",
+        subject: ` Air quality of ${randomFavorite.name} is expected to be ${utils.mapPMValues(pmValue)} with a concentration level of ${pmValue.toFixed(2)}µg/m3!`,
```

Comment:
This is great! Thanks @BenjaminSsempala ....

Just out of curiosity, is this `randomFavorite` attached to that specific user? 

Otherwise, maintenance of our notifications feature is very much welcome! :)

Reply:
Yes @Baalmart  , the favorite is attached to the specific user.